cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/MonitorUtilsTest.java

```diff
@@ -42,30 +43,51 @@ public void testMetadataChanged() {
     Cluster cluster1 = new Cluster("cluster", Arrays.asList(node0, node1, node2), partitions1,
                                    Collections.emptySet(), Collections.emptySet());
 
+    // Verify number of replicas in the cluster
+    assertEquals(8, MonitorUtils.numReplicas(cluster1));
+
```

Comment:
nit: There's no test for brokersWithReplicas(). Maybe we can add the tests here as well?